src/makeCurvedLinks.ts

```diff
@@ -52,6 +54,8 @@ const createCurve = (
   if (i === (inGroup - 1) / 2) {
     return {
       ...link,
+      source,
```

Comment:
are you sure link does not have source/target here?

Reply:
I calcul the new point with the size.

https://github.com/unirakun/react-kgraph/pull/30/files/6c0b90b85a7ce050b3b4bd398c50e6feb5326477#diff-47fe65389c52685395df9cee87630629R51

Reply:
Ok the purpose is to inject the scaled value then.

I don't think this is ok, size is already given if the user needs it.

Reply:
I don't understand why `size` must be use by User ?
It's not better to give all `point` already calculate with size ?

Reply:
Is node exists with x/y scaled? no
So the link does not have to.